### PR TITLE
Added Twig function wp_get_template_part()

### PR DESCRIPTION
**Pull request by joschi127**
Very useful to use custom themes. Would be great if this could be merged. :-)

### CODE REVIEW

**Comment by eko** [comment]
Thank you for this addition, can you please update the ThemeExtensionTest please?

**Comment by joschi127** [comment]
Sorry, did not see that you've covered everything by tests.

I added the test to the ThemeExtensionTest. I hope it is fine like this.

File: Tests/Twig/Extension/ThemeExtensionTest.php [hunk 1]
```diff
@@ -129,5 +140,17 @@ public function testGetFooter()
...
+        /**
+         * Tests the getTemplatePart() method (Twig function: wp_get_template_part()).
+         */
+        public function testGetTemplatePart()
+        {
+            ob_start();
+            $this->extension->getTemplatePart('content', 'page');
+            $content = ob_get_clean();
+
+            $this->assertEquals('wordpress template part content-page', $content);
```
Comment (eko) [inline Tests/Twig/Extension/ThemeExtensionTest.php]: This is failing actually, it should be:

```php
$this->assertEquals('wordpress template part content page', $content);
```

Thank you ;)

**Comment by joschi127** [comment]
As you can see I manually copied & pasted the changes to a clean repo and forgot one of the changed lines in the end ;)

Fixed. But I did it the other way round because "content-page" is the way it is named within Wordpress.

**Comment by eko** [comment]
Thank you @joschi127 :)

**Comment by joschi127** [comment]
Thank you for your quick response and for merging @eko :)

**Pull request merged**